nix/wlroots.nix

```diff
@@ -44,10 +44,7 @@ assert (lib.assertMsg (hidpiXWayland -> enableXWayland) ''
           })
         ])
         ++ (lib.optionals nvidiaPatches [
-          (fetchpatch {
-            url = "https://aur.archlinux.org/cgit/aur.git/plain/0001-nvidia-format-workaround.patch?h=hyprland-nvidia-screenshare-git";
-            sha256 = "A9f1p5EW++mGCaNq8w7ZJfeWmvTfUm4iO+1KDcnqYX8=";
-          })
+          ./wlroots-nvidia.patch
```

Comment:
May I ask what the reason for this change is?

Reply:
AUR patch was:
- out-of-date, so didn't apply after [wlroots@c4d4409deb50](https://gitlab.freedesktop.org/wlroots/wlroots/-/commit/c4d4409deb50) back then (when tested by CI)
- referenced HEAD commit, so may not apply when using older Hyprland (when bisecting)

Reply:
That makes sense, thanks for the explanation.